js/client.js

```diff
@@ -60,20 +61,37 @@ class Client {
   }
 
   // Send a looking-for-game message.
-  register() {
+  register(hasComputerOpponent) {
     this.registered = true; // UI checks this to refresh on game seek
     console.log("registering as " + this.name)
     if (this.game) {
-      this.send({op: "register", name: this.name, seeking: !this.game.started()})
+      this.send({op: "register", name: this.name, seeking: !this.game.started(), hasComputerOpponent:hasComputerOpponent})
```

Comment:
you might just be able to do

this.send({op: "register", name: this.name, seeking: !this.game.started(), hasComputerOpponent})

depending on whether that subset of es6 is supported. i'd try and see